pkg/k8scontext/context.go

```diff
@@ -278,6 +281,15 @@ func (c *Context) ListIstioGateways() []*v1alpha3.Gateway {
 	return gateways
 }
 
+// ListIstioVirtualServices returns a list of discovered Istio Virtual Services
+func (c *Context) ListIstioVirtualServices() []*v1alpha3.VirtualService {
+	var virtualServices []*v1alpha3.VirtualService
+	for _, virtualService := range c.Caches.IstioVirtualService.List() {
+		virtualServices = append(virtualServices, virtualService.(*v1alpha3.VirtualService))
+	}
+	return virtualServices
```

Comment:
It might be very helpful to add something like `glog.V(5).Info("Found VirtualServices: a,b,c,d")` before we exit this

Reply:
(@rheaj99 if you think `glog` here would be helpful - feel free to add these in another PR so we can merge this here)